packages/react-widgets/src/models/utils.js

```diff
@@ -0,0 +1,42 @@
+import { AggregationTypes, _filtersToSQL } from '@carto/react-core';
+
+export function formatTableNameWithFilters(props) {
+  const { source, spatialFilter } = props;
+  const { data, filters, filtersLogicalOperator } = source;
+
+  const filtersClause = _filtersToSQL(filters, filtersLogicalOperator);
+
+  let query = `(${data} ${filtersClause}) foo`;
```

Comment:
I bet we can find a better name than `foo` :)

Reply:
... bar ?